### PR TITLE
 Move page indicator to a proper place 

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] #913
[Problem] Page indicator in incorrect lace
[Solution] Move it to proper position. Update names

### CODE REVIEW

**Comment by HunseopJeong** [comment]
In commit message, typo /lace/place/

File: examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html [hunk 1]
```diff
@@ -7,9 +7,9 @@
 		Wearable UI
 	</title>
 	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
```
Comment (HunseopJeong) [inline examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html]: Leave the  `media="all and (-tizen-geometric-shape: circle)"`

File: examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html [hunk 2]
```diff
@@ -7,9 +7,9 @@
...
 	<link href="../../../css/style.css" rel="stylesheet" />
-	<link href="../../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" rel="stylesheet" />
```
Comment (HunseopJeong) [inline examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html]: ditto

File: examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html [hunk 3]
```diff
@@ -20,19 +20,19 @@
...
-				Indicator
+				Page indicator
 			</h2>
 		</header>
 		<div class="ui-content">
 			<ul class="ui-listview">
 				<li>
-					<a href="pageindicator.html">
-						Linear
+					<a href="pageindicator_circle.html">
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html]: How about renaming pageindicator_circle.html to pageindicator-circle.html?

File: examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html (hunk 1 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html]: ok, I left it, but we need to discuss this globally

File: examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html (hunk 2 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html]: ok

File: examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html (hunk 3 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html]: done

**Review by grzegorz-czajkowski — APPROVED** [review]

**Comment by lslachciak** [comment]
Applied @grzegorz-czajkowski and @HunseopJeong comments
(the last update was with commit message update)

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Review by DohyungLim — APPROVED** [review]
LGTM.

**Pull request merged**